sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_generated/_text_analytics_client.py

```diff
@@ -73,8 +73,8 @@ def __init__(
     ):
         if api_version == 'v3.0':
             base_url = '{Endpoint}/text/analytics/v3.0'
-        elif api_version == 'v3.1-preview.3':
-            base_url = '{Endpoint}/text/analytics/v3.1-preview.3'
+        elif api_version == 'v3.1-preview.4':
```

Comment:
Are we going to bump the version here like we did before for v3.1-preview.4 (i.e. removing support for other API versions)?

Reply:
For now I've added this back in to prevent existing tests from failing.  It will be easier to remove this later once updates to Healthcare and Analyze have been merged, because the tests will be updated to reflect the changes.

Reply:
yeah @abhahn you made the right call. We need to keep v3.1-preview.3 and 4 for now, but once 4 is released we should get rid of 3